### PR TITLE
Implemented Akim, the Soaring Wind, and new CreatedTokenEvent

### PR DESCRIPTION
**Pull request by AsterAether**
I need some feedback on if the way I implemented the CreatedTokenEvent is right, or if it should be handled differently.

### CODE REVIEW

**Comment by JayDi85** [comment]
It's already have CREATE_TOKEN event. Use it instead the new one.

**Comment by AsterAether** [comment]
That event is just for token replacment effects (EG doubling season etc.) it won't trigger if you try using it in a TriggeredAbilityImpl. This "Whenever you create one or more tokens for the first time each turn" is completetly new.

File: Mage.Sets/src/mage/cards/a/AkimTheSoaringWind.java [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+        }
+    }
+
+    @Override
+    public void reset() {
+        playerIds.clear();
+    }
+
+    boolean firstToken(UUID playerId) {
+        return playerIds.get(playerId) == TokenState.FirstToken;
```
Comment (JayDi85) [inline Mage.Sets/src/mage/cards/a/AkimTheSoaringWind.java:147]: `getOrDefault` instead `get`

File: Mage.Sets/src/mage/cards/a/AkimTheSoaringWind.java [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+        this.subtype.add(SubType.BIRD);
+        this.subtype.add(SubType.DINOSAUR);
+        this.power = new MageInt(3);
+        this.toughness = new MageInt(4);
+
+        // Flying
+        this.addAbility(FlyingAbility.getInstance());
+
+        // Whenever you create one or more tokens for the first time each turn, create a 1/1 white Bird creature token with flying.
+
```
Comment (JayDi85) [inline Mage.Sets/src/mage/cards/a/AkimTheSoaringWind.java:52]: Don't need empty line after ability comment

File: Mage/src/main/java/mage/game/events/GameEvent.java [hunk 3]
```diff
@@ -316,6 +316,7 @@ targetId    id if the ability the mana was paid for (not the sourceId)
         LOST_CONTROL,
         GAIN_CONTROL, GAINED_CONTROL,
         CREATE_TOKEN,
+        CREATED_TOKEN,
```
Comment (JayDi85) [inline Mage/src/main/java/mage/game/events/GameEvent.java:319]: at the same line as `CREATE_TOKEN` (see GAIN_CONTROL above)

File: Mage.Sets/src/mage/cards/a/AkimTheSoaringWind.java [hunk 4]
```diff
@@ -0,0 +1,149 @@
...
+        this.addAbility(new AkimTheSoaringTokenAbility(), new AkimTheSoaringWindWatcher());
```
Comment (JayDi85) [inline Mage.Sets/src/mage/cards/a/AkimTheSoaringWind.java:53]: If you add condition hint to card then you can improve card's usability for users. Search for example:
* `.addHint(new ConditionHint(LudevicNecroAlchemistCondition.instance, "Player other than you lost life this turn")));`

How to add:
1. Extract checkTrigger code to Condition object;
2. Use that condition in checkTrigger;
3. Add condition hint.

You can use same condition hints in your another PR (#6430).

**Comment by AsterAether** [comment]
So, that should cover everything

**Pull request merged**